crates/core/src/factories/core_fetch.rs

```diff
@@ -146,6 +165,38 @@ impl FetchFactory for CoreFetchFactory {
 }
 
 type FetchRequest = (OpId, AgentId);
+type FetchResponse = (Vec<OpId>, Url);
+
+#[derive(Debug)]
+struct FetchResponseHandler {
+    response_tx: Sender<FetchResponse>,
+}
+
+impl TxModuleHandler for FetchResponseHandler {
+    fn recv_module_msg(
+        &self,
+        peer: Url,
+        _space: SpaceId,
+        _module: String,
```

Comment:
I don't know what to do with space and module here. For every space there'll be a fetch module instance, and this is a handler registered specifically for the fetch module, so what are these parameters for?

Reply:
Just in case you want to use the same callback for multiple spaces / modules. You can safely ignore them.